### PR TITLE
Release build updates

### PR DESCRIPTION
**Pull request by zsistla**
Added support for PHP 8.1 release to `Makefile` and `make\release`.
Modified documentation.

### CODE REVIEW

**Comment by zsistla** [comment]
Fixes #312

File: make/release.mk [hunk 1]
```diff
@@ -139,6 +141,7 @@ release-$1-zts: Makefile agent | releases/$$(RELEASE_OS)/agent/$$(RELEASE_ARCH)/
 
 endef
```
Comment (mfulb) [inline make/release.mk:142]: I think this should be "endif"?

Comment (zsistla) [reply make/release.mk:142]: When a define includes multiple lines, endef is used to show that the define is ending.
See here for more details: https://www.gnu.org/software/make/manual/html_node/Multi_002dLine.html

**Comment by zsistla** [comment]
Dependent on a hudson PR being merged.  Rerun tests after that is merged.

File: make/release.mk (hunk 1 above)
Comment (mfulb) [reply make/release.mk:142]: Ah thanks. :)

**Review by ZNeumann — APPROVED** [review]

**Comment by mfulb** [comment]
ok jenkins

**Comment by ZNeumann** [comment]
ok jenkins

**Pull request merged**